src/components/LiRepos/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './index.scss';
+
+export default class LiRepos extends PureComponent {
+  static propTypes = {
+    reposData: PropTypes.array,
```

Comment:
since in component namespace, no need to prefix prop like `repoData`, `repoType`, just omit name prefix, use `data`, `type` simply